tools/sync_test_server/start_local_server.sh

```diff
@@ -25,6 +36,12 @@ function echo_step () {
 }
 
 function check_dependencies () {
+  if  ! which -s node; then
+    echo "Error: NodeJS not found. Install using 'brew install node'" && exit 1
+  fi
+  if ! which -s yarn; then
+    echo "Error: Yarn not found. Install using 'brew install yarn'" && exit 1
+  fi
```

Comment:
Moved this before the check for `realm-cli --version` since that requires npm if that check fails.